README.md

```diff
@@ -180,6 +184,12 @@ python3 -m http.server
 +   'https://xayn_ai_staging_assets.s3-de-central.profitbricks.com';
 ```
 
+Then use `cargo make build-web` and `cargo make serve-web` like above.
+
+This only works if the the release repos branch and this branch are in sync
```

Comment:
That the interface must mach is usual stuff and for example `flutter build web` gives error if the api we refer to is broken.
Are any case where the mismatch is not visibile at compile time and gives an error at runtime?

Reply:
There could be not all breaking changes are always visible in the API.

Reply:
But is not different from any other dependency that we have. We can just add that if there is an unexpected runtime error to first check that the used version is compatible with the example.